packages/notebook/src/actions.tsx

```diff
@@ -2138,6 +2139,10 @@ namespace Private {
             });
             return Promise.resolve(false);
           }
+          if (sessionContext.hasNoKernel) {
+            void sessionContextDialogs.selectKernel(sessionContext);
+            break;
```

Comment:
As the function is `runCell` but this will prompt the user without running the cell. I would return false to be consistent.

```suggestion
            return Promise.resolve(false);
```